dir.props

```diff
@@ -485,6 +485,11 @@
       </PropertyGroup>
     </Otherwise>
   </Choose>
+  
+  <!-- Default Test platform to deploy the netstandard compiled tests to -->
+  <PropertyGroup>
+    <TestTFM Condition="'$(TestTFM)'==''>netcoreapp1.0</TestTFM>
```

Comment:
but what happen when you run it locally? not CI not Helix

---
In reply to: [65789923](https://github.com/dotnet/corefx/pull/9129#discussion_r65789923) [](ancestors = 65789923)